libs/product/testing/src/factories/magento/bundle/bundle.factory.ts

```diff
@@ -3,7 +3,7 @@ import { Injectable } from '@angular/core';
 import {
   DaffModelFactory,
 } from '@daffodil/core/testing';
-import { MagentoProductTypeEnum, MagentoBundledProduct } from '@daffodil/product';
+import { MagentoProductTypeEnum, MagentoBundledProduct } from '@daffodil/product/driver/magento';
```

Comment:
these magento factories should go in `@daffodil/product/driver/magento/testing`

Reply:
moved